src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -5,6 +5,7 @@ export class FilterOption extends HTMLElement {
     this.viewCheckboxLabels = new Set();
     // It contains the checked checkboxes
     this.activeCheckboxLabels = new Set();
+    this.otherLabel = 'other';
```

Comment:
To emphasize that this is an immutable value, use `const FILTER_OPTION_UNDEFINED_LABEL = 'other';` (or a shorter name), with a brief comment before it to explain what it's used for.

Reply:
Would I declare it in constructor? Or declare it as `const` inside the methods?

Reply:
Outside the class.

Reply:
updated :-)